project.py

```diff
@@ -114,6 +134,79 @@ def create_trackbars(self):
         cv.createTrackbar('lo', 'result', self.lo, 255, self.trackbar_callback)
         cv.createTrackbar('hi', 'result', self.hi, 255, self.trackbar_callback)
 
+    def check_change(self, image, mask):
+        imageOld = cv.imread("slide"+repr(self.numberOfSlides)+".png",
+                                cv.CV_LOAD_IMAGE_COLOR)
+        maskOld = cv.imread("mask.png", cv.CV_LOAD_IMAGE_GRAYSCALE)
+        if(maskOld is None):
+            return True
+
+        if(mask.shape != maskOld.shape):
+            return False
+
+        applyMask = cv.bitwise_and(maskOld, mask)
+        imageOld = cv.bitwise_and(imageOld, imageOld, mask=applyMask)
+        checkImage = cv.bitwise_and(image, image, mask=applyMask)
+
+        orb = cv.ORB()
+        kp1, ds1 = orb.detectAndCompute(imageOld, None)
```

Comment:
@Foxiy did you profile this? I'd guess that `ORB` is pretty expensive in terms of computational resources.